### PR TITLE
Log Stacktraces.

### PR DESCRIPTION
**Pull request by danielfenner**
Was helpful for debugging tests. Just uses absl.

### CODE REVIEW

File: src/OrbitBase/Logging.cpp [hunk 1]
```diff
@@ -28,3 +33,17 @@ void LogToFile(const std::string& message) {
     log_file.flush();
   }
 }
+
+void LogStacktrace() {
+  constexpr int kMaxDepth = 64;
+  std::vector<void*> raw_stack(kMaxDepth);
+  const int raw_stack_size = absl::GetStackTrace(&raw_stack[0], kMaxDepth, 1);
+  for (int i = 0; i < raw_stack_size; ++i) {
+    char tmp[1024];
```
Comment (dimitry-) [inline src/OrbitBase/Logging.cpp:42]: std::array<>

File: src/OrbitBase/Logging.cpp [hunk 2]
```diff
@@ -28,3 +33,17 @@ void LogToFile(const std::string& message) {
     log_file.flush();
   }
 }
+
+void LogStacktrace() {
+  constexpr int kMaxDepth = 64;
```
Comment (dimitry-) [inline src/OrbitBase/Logging.cpp:38]: size_t

**Review by dimitry- — COMMENTED** [review]
I wonder if this should go to Debug(Logging).h

**Review by dimitry- — APPROVED** [review]
otherwise lgtm.

File: src/OrbitBase/Logging.cpp (hunk 1 above)
Comment (danielfenner) [reply src/OrbitBase/Logging.cpp:42]: done

File: src/OrbitBase/Logging.cpp (hunk 2 above)
Comment (danielfenner) [reply src/OrbitBase/Logging.cpp]: done

**Pull request merged**